machine-learning/lambda-v2/README.md

```diff
@@ -127,4 +135,4 @@ Follow the steps in order to prepare the component artifacts, recipes and create
 
 ## Deploy components
 
-Follow the [documentation](https://docs.aws.amazon.com/greengrass/v2/developerguide/get-started-with-edge-manager-on-greengrass.html#run-sample-sme-image-classification-inference) to deploy inference and model components and run inference on the edge using sagemaker edge manager. 
\ No newline at end of file
+Follow the [documentation]() to deploy inference and model components and run inference on the edge using lambdas in Greengrassv2. 
```

Comment:
Should we remove this line until the tutorial is available in the documentation?